exe/sufia_import

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+require 'rails'
+require ::File.join(File.dirname(Bundler.default_gemfile), 'config/environment')
+
+options = OpenStruct.new
+options.verbose = false
+options.import_dir = "tmp/import"
+# TODO: add collection like options.mappings = ["Collection=collection_", "GenericFile=generic_file_"]
+options.mappings = ["GenericFile=generic_file_"]
+
+args = OptionParser.new do |opts|
+  opts.banner = "imports model metdata json files in, contacting fedora to import binaries. Usage: sufia_import [options]"
+
+  opts.on("-v", "--[no-]verbose", "Run verbosely") do |v|
+    options.verbose = v
+  end
+
+  opts.on("-d", "--json_directory [path/to/files]", "Directory containing files for import.  Defaults to tmp/import") do |dir|
+    options.import_dir = dir
+  end
+
+  opts.on("--json_mapping [class=prefix, ...]", Array, "For each class: prefix pair, import the files discovedrable by the prefix as objects ot the class. Defaults to Collection=collection_, GenericFile=generic_file_ ") do |mappings|
+    options.mappings = mappings
+  end
+end
+
+begin
+  args.order!
+  args.parse!
+rescue => error
+  $stderr.puts "ERROR: #{error}\n"
+  $stderr.puts args.help
+  exit 1
+end
+
+verbose = options[:verbose]
+$stdout.puts "running with #{options}" if verbose
+
+options.mappings.each do |pair|
+  class_name, prefix = pair.split('=')
+  translator = "Sufia::Import::#{class_name}Translator".constantize.new(sufia6_user: 'username', sufia6_password: 'password', import_binary: 'false')
```

Comment:
I'm wondering how extensible this is.  I know it matches the ticket, but if I wanted to add my own ticket.  How about if we change it to just the translator class and the file pattern?  
`Sufia::Import::GenericFileTranslator=generic_file_,Sufia::Import::CollectionTranslator=collection_`
That way other people could namespace and name the translators as they please...